mobilecoind/src/payments.rs

```diff
@@ -982,15 +987,22 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
         }
         let change = input_value - total_value - tx_builder.get_fee();
 
-        // If we do, add an output for that as well.
+        // If we do have nonzero change, add an output for that as well.
         // TODO: Should the exchange write destination memos?
```

Comment:
Not in your PR - but wondering if we have this TODO captured as a ticket anywhere?

Reply:
Yes there is a GitHub issue

Reply:
#1522

Reply:
Ty!!